### PR TITLE
📖 add release-0.5 to README.md

### PR DESCRIPTION
**Pull request by tuminoid**
Add release-0.5 information to README.md

### CODE REVIEW

**Comment by tuminoid** [comment]
/override test-ubuntu-integration-main test-centos-e2e-integration-main metal3-bmo-e2e-test 
Documentation change.

File: README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ development. Here are the tests which run on different branches in BMO:
 * **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
   main branch and BMO **main** branch on Ubuntu
 
+**Release-0.5 branch:**
+
+* **/test-centos-e2e-integration-release-1-6** runs CAPM3 e2e integration tests
+  with CAPM3 release-1.5 branch and BMO **release-0.5** branch on Centos
+* **/test-ubuntu-integration-release-1-6** runs ansible integration tests with
+  CAPM3 release-1.5 branch and BMO **release-0.5** branch on Ubuntu
```
Comment (smoshiur1237) [inline README.md:54]:
```suggestion
  CAPM3 release-1.6 branch and BMO **release-0.5** branch on Ubuntu
```

File: README.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ development. Here are the tests which run on different branches in BMO:
 * **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
   main branch and BMO **main** branch on Ubuntu
 
+**Release-0.5 branch:**
+
+* **/test-centos-e2e-integration-release-1-6** runs CAPM3 e2e integration tests
+  with CAPM3 release-1.5 branch and BMO **release-0.5** branch on Centos
```
Comment (smoshiur1237) [inline README.md:52]:
```suggestion
  with CAPM3 release-1.6 branch and BMO **release-0.5** branch on Centos
```

**Review by smoshiur1237 — COMMENTED** [review]
Nit

File: README.md (hunk 1 above)
Comment (tuminoid) [reply README.md:54]: Fixed

File: README.md (hunk 2 above)
Comment (tuminoid) [reply README.md:52]: Fixed.

**Comment by tuminoid** [comment]
/override test-ubuntu-integration-main test-centos-e2e-integration-main metal3-bmo-e2e-test
Documentation change.

**Review by smoshiur1237 — COMMENTED** [review]
LGTM

**Pull request merged**